config/config.exs

```diff
@@ -60,6 +60,10 @@ config :peerage, log_results: false
 
 config :statix, prefix: "ret"
 
+config :ret,
+  ecto_repos: [Ret.Repo, Ret.SessionLockRepo],
+  turkey_mode?: System.get_env("TURKEY_MODE") === "1"
+
```

Comment:
Could you move this up to replace line 9? While this technically works, it overwrites the effects of line 9 and could be a pain-in-the-butt to find this far down.

Reply:
done